client/js/services/token.js

```diff
@@ -8,14 +8,23 @@ angular
     getToken: () => (
       $window.localStorage.getItem('token')
     ),
-    clearToken: () => {
-      $window.localStorage.removeItem('token');
-    },
-    saveUser: (user) => {
-      $window.localStorage.setItem('user', user);
-    },
-    clearUser: () => {
-      $window.localStorage.removeItem('user');
-    }
+    clearToken: () => (
+      $window.localStorage.removeItem('token')
+    ),
+    saveUserId: userId => (
```

Comment:
added a comment on Yamen's PR about having several getters and setters here.
We can save the entire user object to localStorage, and use (JSON.parse(user)).userId to retrieve them again